src/Apps/__tests__/Fixtures/Artwork/MutationResults.ts

```diff
@@ -22,6 +22,18 @@ export const OrderWithFailure = {
   },
 }
 
+export const OrderWithMissingInfo = {
```

Comment:
Looks like this is not used anywhere?

Reply:
Whoops good catch!

Reply:
@lilyfromseattle are you planning on removing this bit, or is it needed?

Reply:
@pepopowitz Already removed just forgot to push. Should be good to go now.